assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -507,6 +518,7 @@ function Carousel() {
           </MenuIconsWrapper>
         </MenuArea>
       </Wrapper>
+      <EditorVersion>{`version ${version}`}</EditorVersion>
```

Comment:
This needs to be translatable. Using `__()` and `sprintf()`.

Reply:
Woops, good catch thanks!